beacon-chain/db/kv/archived_point.go

```diff
@@ -41,6 +41,67 @@ func (k *Store) SaveArchivedPointRoot(ctx context.Context, blockRoot [32]byte, i
 	})
 }
 
+// SaveLastArchivedIndex to the db.
+func (k *Store) SaveLastArchivedIndex(ctx context.Context, index uint64) error {
```

Comment:
Would this be last or latest?

Reply:
It's the last one. Latest one could work too but I'm inclined to keep it as last : )